unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -138,19 +138,21 @@ class ConfigureHUDElementMessage
     {
         public HUDElementID hudElementId;
         public HUDConfiguration configuration;
+        public string extraPayload;
     }
 
     public void ConfigureHUDElement(string payload)
     {
         ConfigureHUDElementMessage message = JsonUtility.FromJson<ConfigureHUDElementMessage>(payload);
 
-        HUDConfiguration configuration = message.configuration;
         HUDElementID id = message.hudElementId;
+        HUDConfiguration configuration = message.configuration;
+        string extraPayload = message.extraPayload;
 
-        ConfigureHUDElement(id, configuration);
+        ConfigureHUDElement(id, configuration, extraPayload);
     }
 
-    public void ConfigureHUDElement(HUDElementID hudElementId, HUDConfiguration configuration)
+    public void ConfigureHUDElement(HUDElementID hudElementId, HUDConfiguration configuration, string extraPayload)
```

Comment:
I would have this extrapayload defaulted to null, to avoid boilerplate where the most likely case is gonna be null

Reply:
It comes defaulted as null from kernel
`public ConfigureHUDElement(hudElementId: HUDElementID, configuration: HUDConfiguration, extraPayload: any | null = null)`
`extraPayload: extraPayload ? JSON.stringify(extraPayload) : null`

Reply:
I meant as a default param `string extraPayload = null`

Reply:
ups sorry, I misunderstand you, done.